lib/commands/session.js

```diff
@@ -13,6 +13,7 @@ commands.createSession = async function (caps) {
   this.validateDesiredCaps(caps);
   this.sessionId = UUID.create().hex;
   this.caps = caps;
+  this.rawCaps = _.cloneDeep(caps); // used for reset
```

Comment:
We copy initialOpts 3 line below, but they are not caps. Maybe we should merge the 2.